### DISCUSSION TITLE
Error Messages

### DISCUSSION THREAD

**Discussion by milkias17** [NeogitOrg/neogit#1051] [Q&A]
Is there a way to get the actual error messages pop up like they used to in the "old" neogit. Right now if an error occurs, it just says an error occurred, any configuration or etc to get the actual error?

**Comment by CKolkey** [comment]
If it's for a CLI command that was run, you can open the git command history buffer via `$` and see the output of a command by hitting `<tab>` on the specific call. Is that what you're looking for?

**Reply by milkias17** [reply]
Thanks🙏. But was looking for the previous way neogit used to actually send the error of the git cli as a notification, and was wondering if I could get that back?

**Reply by milkias17** [reply]
Hey👋, was wondering if a PR might be welcome for a config option to enable that, if it was removed totally? Really sorry for the nag😅, I really loved that feature.

Thanks a lot for everything you've done to the project, makes using neovim a much much better experience.

**Reply by CKolkey** [reply]
Yea, PR's are always welcome :)

**Reply by milkias17** [reply]
thanks for the reply🙏.

**Reply by CKolkey** [reply]
Of course. Lemme know if you have any questions about the config stuff :)
